dataactbroker/handlers/accountHandler.py

```diff
@@ -122,7 +123,7 @@ def login(self,session):
                 raise ve
             except Exception as e:
                 LoginSession.logout(session)
-                raise ValueError("Invalid username and/or password")
+                raise e
```

Comment:
Curious -- do we know what this exception might be? Does changing the exception type affect anything downstream?